src/main/kotlin/no/skatteetaten/aurora/mokey/controller/StorageGridObjectAreaController.kt

```diff
@@ -0,0 +1,23 @@
+package no.skatteetaten.aurora.mokey.controller
+
+import no.skatteetaten.aurora.mokey.model.StorageGridObjectAreaDetails
+import no.skatteetaten.aurora.mokey.service.StorageGridObjectAreaService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(StorageGridObjectAreaController.path)
+class StorageGridObjectAreaController(
+    val storageGridObjectAreaService: StorageGridObjectAreaService
+) {
+    @GetMapping
+    suspend fun getAll(@RequestParam affiliation: String): List<StorageGridObjectAreaDetails> {
```

Comment:
jeg lurer på om kanskje query parameteren affiliation bør være en liste. Jeg har sett at det er det for applicationControlleren. Det gjør det mer versatilt i gobo når man bruker graphql med dette

Reply:
ref. pm. vi beholder det som det er og endrer ved behov